skfda/tests/test_stats.py

```diff
@@ -144,3 +155,48 @@ def test_mei(self) -> None:
             ]),
             rtol=1e-5,
         )
+
+
+class TestStd(unittest.TestCase):
```

Comment:
We are trying to go with Pytest-style tests for new tests, instead of unittest-style ones. Please take a look at them.